scripts/README_APK_WRAPPER.md

```diff
@@ -0,0 +1,268 @@
+# Wire APK Wrapper Script
+
+This script automates the process of modifying Wire APK files by moving the `com/wire` folder from `smali` to `smali_classes2` and rebuilding the APK.
+
+## What It Does
+
+1. **Decompiles** the Wire APK using apktool
+2. **Moves** the `com/wire` folder from `smali` to `smali_classes2`
+3. **Rebuilds** the APK with apktool
+4. **Renames** the output APK with `_wrapped` suffix
+5. **Optionally signs** the APK for installation (if `--sign` option is used)
+
+## Prerequisites
+
+### Required Tools
+
+#### 1. Java Development Kit (JDK)
+- **Required**: Java 11 or higher
+- **Download**: [OpenJDK](https://openjdk.org/)
+- **Purpose**: Needed for running apktool and jarsigner
+- **Installation**:
+  - **Windows**: Download installer from Oracle/OpenJDK website
+  - **macOS**: `brew install openjdk` or download from website
+  - **Linux**: `sudo apt install default-jdk` (Ubuntu/Debian) or `sudo yum install java-1.8.0-openjdk` (CentOS/RHEL)
```

Comment:
Personally, I wouldn't add these recommendations here.

1. Windows is probably easier using `winget` instead of downloading stuff from _Oracle_. Also... Open.
2. There's `sdkman` for Linux.

Better to just remove these.